scripts/eia/doi_Nigeria-ATAFI-Validations.R

```diff
@@ -18,20 +18,17 @@ carob_script <- function(path) {
    
    meta <- data.frame(
       # Need to fill-in metadata...
-      # carobiner::read_metadata(uri, path, group, major=2, minor=0),
-      uri = carobiner::simple_uri(uri),
-      dataset_id = uri,
       authors = "Mary Jane;John Do",
-      data_citation="...",
-      data_institute = NA ,
+      data_institute = "AAA-BBB" , # partner name 
       title = NA,
       description = "Nigeria-ATAFI-Validations",
       group = group,
       publication=NA,
-      license = 'Some license here...',
+      use_case = NA,
```

Comment:
The use case is `WA-Rice-ATAFI/MOVE` on [terms/values/values_eia_usecases.csv#L8](https://github.com/EiA2030/eia-carob/blob/6583263029293c52ba7d574b2536c1a31c9db70d/terms/values/values_eia_usecases.csv#L8)